frontend/src/routes.ts

```diff
@@ -0,0 +1,14 @@
+export const TransactionsLocation = "/transactions";
```

Comment:
Siūlyčiau šitą failą pavadint `route.constants.ts` bet čia gal personal preference

Reply:
Blem, gera ideja